scripts/irods/test/test_irsync.py

```diff
@@ -772,3 +772,35 @@ def test_irsync_with_all_keyword_does_not_fail_on_overwrite_issue__issue_8295(se
             f.write('potato')
 
         self.user0.assert_icommand(['irsync', '-a', test_file_path, f'i:{test_data_object_path}'])
+
+    def test_irsync_with_all_keyword(self):
+        data_object_name = 'all_keyword_dataobj'
+        another_resc  = 'resc_all_keyword'
+        another_resc_dir = os.path.join(self.admin.local_session_dir, 'vault_all_keyword')
+        data_object_path = f'{self.admin.session_collection}/{data_object_name}'
+
+        test_file_path = os.path.join(self.testing_tmp_dir, data_object_name)
+
+        with open(test_file_path, 'w') as f:
+            f.write('potato')
+
+        self.admin.assert_icommand(['itouch', data_object_path])
```

Comment:
Would be better if we performed the non-admin-y things as a non-rodsadmin, but certainly not required.

Reply:
I agree. Let's use the non-admin account instead.